pk_tic_tac_toe.rb

```diff
@@ -83,42 +87,32 @@ def check_win(board, player, victory)
   end
 end
 
-def play_turn(player, board, victory)
-  print "#{player.name}, it's your turn; "
-  make_move(board, player)
+def play_turn(player, board)
   display_board(board)
```

Comment:
Show `display_board` method before `make_move` method, which prompts the user to enter their board position.